mantle/cmd/kola/testiso.go

```diff
@@ -202,6 +202,10 @@ var nmConnectionFile = "coreos-dhcp.nmconnection"
 var nmConnection = fmt.Sprintf(`[connection]
 id=%s
 type=ethernet
+# add wait-device-timeout here so we make sure NetworkManager-wait-online.service will
+# wait for a device to be present before exiting. See
+# https://github.com/coreos/fedora-coreos-tracker/issues/1275#issuecomment-1231605438
+wait-device-timeout=20000
```

Comment:
Hmm I'm also confused why we even need this and have the same question as you in https://github.com/coreos/fedora-coreos-tracker/issues/1275#issuecomment-1231605438.

Reply:
yeah I do think this is acting as designed (i.e. from the description in the [linked page](https://networkmanager.dev/docs/api/latest/NetworkManager-wait-online.service.html)) but maybe not.